website/src/components/Showcase/Showcase.js

```diff
@@ -6,20 +6,22 @@ import styles from './Showcase.module.scss';
 import _ from 'lodash';
 import { TailSpin } from 'react-loader-spinner';
 
-function Showcase() {
+const Showcase = () => {
   const [cards, setCards] = useState([]);
-  const [toRender, setToRender] = useState([]);
+  const [toRender, setToRender] = useState([<TailSpin />]);
 
   useEffect(() => {
-    setCards(_makeShuffledCardList(cardList).map(_makeCard));
+    const shouldShuffle = new URLSearchParams(window.location.search).get('shuffle') !== 'disabled';
+    const resultingCardList = shouldShuffle ? _makeShuffledCardList(cardList) : cardList;
+    setCards(resultingCardList.map(_makeCard));
   }, []);
 
   useEffect(() => {
```

Comment:
```suggestion
return cards.length > 0 ? _makeCardsContainer(cards) : _makeLoadingSpinner();
```

Could you explain why do we need such advanced manipulations with that `toRender` and `setToRender` state? Why can't we just return those values in a simple return statement?

Reply:
There was a spinner loader implemented to the Showcase page that added a `toRender `and `setToRender` state.  I will work on simplifying the code to just return the desired component based on the length of the cards array without the intermediate state there.